Security/src/RedisDataProtection/Pages/Index.cshtml

```diff
@@ -9,6 +9,28 @@
     <p>Learn about <a href="https://learn.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
 </div>
 
+<!-- Steeltoe: Display XML that contains crypto keys used by Data Protection API. -->
+<div class="alert alert-secondary">
+    <h4>Crypto keys used by Data Protection API</h4>
+    <table class="table table-bordered table-primary" style="display: table; table-layout: fixed; width: 100%;">
+        <thead>
+        <tr>
+                <th scope="col">Crypto Key XML</th>
+        </tr>
```

Comment:
```suggestion
            <tr>
                <th scope="col">Crypto Key XML</th>
            </tr>
```

Reply:
I've reformatted the file using Rider. Apparently, it puts `<tr>` at the same indent depth as `<thead>`. I'm indifferent, just aligning with how it already was in other places.